boards/BMS/Inc/Io/Io_Temperatures.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include "App_SharedExitCode.h"
+
+/**
+ * Read internal die temperatures for all cell monitoring chips.
+ * @return EXIT_CODE_OK if internal die temperatures were acquired successfully
+ * from all cell monitoring chips. Else, EXIT_CODE_ERROR.
+ */
+ExitCode Io_Temperatures_ReadDieTemperatures(void);
+
+/**
+ * Get the pointer to the array containing internal die temperatures read
+ * from the cell monitoring chips.
+ * @note The subscript value of the array is the number of cell monitoring
+ * devices on the daisy chain.
+ * @note Call Io_Temperatures_ReadDieTemperatures to acquire the most recent
+ * die temperatures from the cell monitoring chips before calling this function.
+ * @note The internal die temperature is read back from the cell monitoring
+ * chips as a voltage. Use the following equation to convert voltages to
+ * temperatures (°C):
+ *
+ * ITMP(Measured_V) = Measured_V * (1°C / 13.1578e-3V) - 276°C
+ *
+ * @return A pointer to the array containing the internal die temperatures for
+ * each cell monitoring chip.
```

Comment:
We should probably note the units for temperature, as well as how long the array is here.

Reply:
Just a question regarding the units for temperature.  Should I mention that I'm returning voltages that have not yet been converted to temperatures?

Reply:
Or should I just mention the unit of temperature that we will convert these voltages to?

Reply:
Or one thing I could do is just store the converted temperature values in a array of floats. This will reduce performance but it's probably negligible for something running in a 1Hz thread. Letting the user know and communicating clearly that we're getting degC seems like a better approach imo.

Reply:
Changed the array of uint16s to an array of floats to store the internal die temperature measurements. Lemme know if you think this is a good idea

Reply:
^I like this better, would have assumed from the previous implementation that it was already in a unit of temperature. Could still add that the unit is [°C]